react/OrderItems.tsx

```diff
@@ -19,9 +19,13 @@ import { useOrderForm } from 'vtex.order-manager/OrderForm'
 const DEBOUNCE_TIME_MS = 300
 
 const AVAILABLE = 'available'
-const SUBTOTAL_TOTALIZER_ID = 'Items'
 const TASK_CANCELLED = 'TASK_CANCELLED'
 
+enum Totalizers {
+  SUBTOTAL = 'Items',
+  DISCOUNT = 'Discounts',
+}
+
```

Comment:
why aren't you using `react-intl`?

Reply:
Those are the IDs from the totalizers that come from the API and are used solely to identify the corresponding fields of the `totalizers` object of the OrderForm. No need to translate them, hehe.

Reply:
really, my bad.

Reply:
That was a perfectly valid question. I guess the point of reviewing a code you have little context about is that you catch stuff that other people might not notice. Thanks for the review!